po/fi.po

```diff
@@ -1430,7 +1429,7 @@ msgstr "Yritys"
 #: gnucash/gnome/dialog-job.c:594 gnucash/gnome/dialog-vendor.c:742
 #: gnucash/gnome-utils/gnc-tree-view-owner.c:349
 msgid "ID #"
-msgstr "ID nro"
+msgstr "ID #"
```

Comment:
The english use "#" as abbreviation for "Number" in forms, but in other laguages we use "Nr.", "No.", ...